vshard/storage/init.lua

```diff
@@ -1196,20 +1207,28 @@ local function recovery_f()
             goto sleep
         end
 
+        service:set_activity('recovering sending buckets')
         ok, total, recovered = pcall(recovery_step_by_type,
                                      consts.BUCKET.SENDING)
         if not ok then
             is_all_recovered = false
-            log.error('Error during sending buckets recovery: %s', total)
+            local err_msg = string.format(
+                'Error during sending buckets recovery: %s', total)
+            service:set_status_error(err_msg)
+            log.error(err_msg)
```

Comment:
Seeing how this patter repeats, maybe it would be nice to make `set_status_error` take the format args and return the resulting string. Like this:
```Lua
log.error(service:set_status_error(
    'Error during sending buckets recovery: %s', total))
```

Reply:
Wow, that looks really convenient. Let's do that